resources/user.rb

```diff
@@ -0,0 +1,106 @@
+property :username, String, name_attribute: true, desired_state: false, identity: true
+property :password, String, desired_state: false, identity: true, sensitive: true
+property :first_name, String, desired_state: false, identity: true, default: ''
+property :last_name, String, desired_state: false, identity: true, default: ''
+property :email, String, desired_state: false, identity: true, default: ''
+property :roles, Array,  desired_state: false, identity: true, default: []
+property :api_endpoint, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['endpoint'] }
+property :api_username, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['username'] }
+property :api_password, String, desired_state: false, identity: true, sensitive: true,
```

Comment:
Once #37 is done, we'll need to replace the 3 above properties :)

Reply:
ack